packages/clay-layout/src/Container.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Adds `.container-fluid` class to create a fluid container that
+	 * doesn't expand beyond a set width
+	 */
+	fluid?: boolean;
```

Comment:
I lost track of where, but we went through a back and forth over `To Fluid or Not To Fluid` and concluded that we didn't want To Fluid 😂 

So, by default, we'd have `<Container>...</Container>` output:

```html
<div class="container-fluid container-fluid-xl">
    ...
</div>
```

This is somewhat opinionated... but so is our design language and as long as this can be overridden we should be fine. Not sure we have a valid case for just `container`, at the very least is not widespread enough to grant a first-level API

Reply:
Hmm, I don't think I was privy to that convo.

IMO, I think its really odd to default to `container-fluid` for a component called `Container`, especially when the class `container` is also a thing. 

I think it'd be fine to keep `container-fluid` as a default for the a taglib in portal, but for the Clay library I don't think it would make sense to default to fluid for the name `Container`

Reply:
Initially, my concern about the using of `container` when we already have a class called `container` was the same @bryceosterhaus already has wrote, after some days taking a look to what lexicon want and what we finally use, the uses cases are only a few, majority of them are the classes `container-fluid container-fluid-max-xl` and we don't use the `standard` bootstrap container.
So for us and right now the default container is the class combination: `container-fluid container-fluid-max-xl`. It still sounds bad to me but I have to admit our container is a no very semantic combination of classes,  maybe it's something we can study and try to improve in a future more semantic CSS.

IMHO, the `<Container>` output should be our standard and more used container